### PR TITLE
Add request balance by address to kaspactl

### PR DESCRIPTION
**Pull request by FestinaLente666**

### CODE REVIEW

File: infrastructure/network/netadapter/server/grpcserver/protowire/README.md [hunk 1]
```diff
@@ -3,13 +3,13 @@ protowire
...
    https://github.com/protocolbuffers/protobuf/releases/download/v3.12.3/protoc-3.12.3-linux-x86_64.zip
-2. `go get github.com/golang/protobuf/protoc-gen-go`
-3. `go get google.golang.org/grpc/cmd/protoc-gen-go-grpc`
+2. `go install github.com/golang/protobuf/protoc-gen-go`
+3. `go install google.golang.org/grpc/cmd/protoc-gen-go-grpc`
 4. In the protowire directory: `go generate .`
 
 Documentation
--------------
+-------------r
```
Comment (stasatdaglabs) [inline infrastructure/network/netadapter/server/grpcserver/protowire/README.md:11]: This appears to be a typo

File: infrastructure/network/netadapter/server/grpcserver/protowire/messages.proto [hunk 2]
```diff
@@ -121,6 +121,8 @@ message KaspadMessage {
     NotifyVirtualDaaScoreChangedRequestMessage notifyVirtualDaaScoreChangedRequest = 1074;
     NotifyVirtualDaaScoreChangedResponseMessage notifyVirtualDaaScoreChangedResponse = 1075;
     VirtualDaaScoreChangedNotificationMessage virtualDaaScoreChangedNotification = 1076;
+    GetBalanceByAddressRequestMessage getBalanceByAddressRequestMessage = 1077;
+    GetBalanceByAddressResponseMessage getBalanceByAddressResponseMessage = 1078;
```
Comment (stasatdaglabs) [inline infrastructure/network/netadapter/server/grpcserver/protowire/messages.proto:125]: For consistency's sake, please remove the "Message" suffix from the variable names

**Review by stasatdaglabs — DISMISSED** [review]

File: infrastructure/network/netadapter/server/grpcserver/protowire/README.md (hunk 1 above)
Comment (FestinaLente666) [reply infrastructure/network/netadapter/server/grpcserver/protowire/README.md]: Fixed

File: infrastructure/network/netadapter/server/grpcserver/protowire/messages.proto (hunk 2 above)
Comment (FestinaLente666) [reply infrastructure/network/netadapter/server/grpcserver/protowire/messages.proto]: Fixed

**Review by michaelsutton — APPROVED** [review]

**Pull request merged**